content/academy/apify_platform/getting_started/apify_client.md

```diff
@@ -0,0 +1,200 @@
+
+```shell
+npm install apify-client
+```
+
+In Python, you can install it from PyPI with this command:
+
+```shell
+pip install apify-client
+```
+
+After installing the package, let's make a file named **client** and import the Apify client like so:
+
+```marked-tabs
+<marked-tab header="Node.js" lang="javascript">
+// client.js
+import { ApifyClient } from 'apify-client';
+</marked-tab>
+<marked-tab header="Python" lang="python">
+# client.py
+from apify_client import ApifyClient
+</marked-tab>
+```
+
+## [](#running-an-actor) Running an actor
+
+In the last lesson, we ran the **adding-actor** and retrieved its dataset items. That's exactly what we're going to do now; however, by using the Apify client instead.
+
+Before we can use the client though, we must create a new instance of the `ApifyClient` class and pass it our API token from the [**Integrations** page](https://console.apify.com/account?tab=integrations) on the Apify Console:
+
+```marked-tabs
+<marked-tab header="Node.js" lang="javascript">
+const client = new ApifyClient({
+    token: 'YOUR_TOKEN',
+});
+</marked-tab>
+<marked-tab header="Python" lang="python">
+client = ApifyClient(token='YOUR_TOKEN')
+</marked-tab>
+```
+
+> If you are planning on publishing your code to a public Github/Gitlab repository or anywhere else online, be sure to set your API token as en environment variable, and never hardcode it directly into your script.
+
+Now that we've got our instance, we can point to an actor using the `client.actor()` function, then call the actor with some input with the `.call()` function.
+
+```marked-tabs
+<marked-tab header="Node.js" lang="javascript">
+const run = await client.actor('YOUR_USERNAME/adding-actor').call({
+    num1: 4,
+    num2: 2,
+});
+</marked-tab>
+<marked-tab header="Python" lang="python">
+run = client.actor('YOUR_USERNAME/adding-actor').call(run_input={
+    'num1': 4,
+    'num2': 2
+})
+</marked-tab>
+```
+
+## [](#collecting-dataset-items) Collecting dataset items
+
+Once an actor's run has completed, it will return a **run info** object that looks something like this:
+
+![Run info object]({{@asset apify_platform/getting_started/images/run-info.webp}})
+
+The `run` variable we created in the last section points to the **run info** object of the run we created with the `.call()` function, which means tha through this variable, we can access the run's `defaultDatasetId`. This ID can then be passed into the `client.dataset()` function.
+
+```marked-tabs
+<marked-tab header="Node.js" lang="javascript">
+const dataset = client.dataset(run['defaultDatasetId'])
+</marked-tab>
+<marked-tab header="Python" lang="python">
+dataset = client.dataset(run['defaultDatasetId'])
+</marked-tab>
+```
+
+Finally, we can collect the items in the dataset by using the `dataset.downloadItems()` function in Node.js, and the `dataset.list_items()` function in Python, then log them to the console.
```

Comment:
If you just want to log the items, you can use the `listItems()` function. It gives you an already parsed JSON array with the items. The download items exists to get a CSV, Excel and so on.